docs/UserGuide.adoc

```diff
@@ -71,6 +71,11 @@ Examples:
 Shows a list of all persons in the address book. +
 Format: `list`
 
+== Sorting all persons : `sort`
+
+Shows a sorted list of all persons in the address book. +
+Format: `sort`
```

Comment:
Nice to see that you have updated the user guide :)

However, do you think a first-time user would know how the list is sorted by when he/she reads this?